api/controller/xtra_db_cluster_api.proto

```diff
@@ -20,6 +20,8 @@ service XtraDBClusterAPI {
   rpc DeleteXtraDBCluster(DeleteXtraDBClusterRequest) returns (DeleteXtraDBClusterResponse);
   // RestartXtraDBCluster restarts XtraDB cluster.
   rpc RestartXtraDBCluster(RestartXtraDBClusterRequest) returns (RestartXtraDBClusterResponse);
+  // GetXtraDBCluster gets a cluster's connection credentials.
+  rpc GetXtraDBCluster(GetXtraDBClusterRequest) returns (GetXtraDBClusterResponse);
```

Comment:
I left comments only about PSMDBClusterAPI assuming that you will do the same for XtraDBClusterAPI. Please do